Mage.Sets/src/mage/cards/e/EzioAuditoreDaFirenze.java

```diff
@@ -0,0 +1,133 @@
+
+    private static final FilterNonlandCard filter = new FilterNonlandCard("assassin spells you cast");
+
+    static {
+        filter.add(SubType.ASSASSIN.getPredicate());
+        filter.add(Predicates.not(new AbilityPredicate(FreerunningAbility.class))); // So there are not redundant copies being added to each card
+    }
+    
+    public EzioAuditoreDaFirenze(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{B}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.ASSASSIN);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Menace
+        this.addAbility(new MenaceAbility(), makeWatcher());
+
+        // Assassin spells you cast have freerunning {B}{B}.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new GainAbilityControlledSpellsEffect(new FreerunningAbility("{B}{B}"), filter)));
+        // Whenever Ezio deals combat damage to a player, you may pay {W}{U}{B}{R}{G} if that player has 10 or less life. When you do, that player loses the game.
+        //Phage this.addAbility(new DealsCombatDamageToAPlayerTriggeredAbility(new LoseGameTargetPlayerEffect(), false, true));
+
+        //this.addAbility(new ConditionalTriggeredAbility(new DealsCombatDamageToAPlayerTriggeredAbility(new EzioAuditoreDaFirenzeEffect(), true, true)), new LifeLossOtherFromCombatWatcher());
+        //this.addAbility(new ConditionalTriggeredAbility(new DealsCombatDamageToAPlayerTriggeredAbility(new DoIfAnyNumberCostPaid(new EzioAuditoreDaFirenzeEffect(),new ManaCostsImpl<>("{W}{U}{B}{R}{G}")), true, true),condition, "pay {W}{U}{B}{R}{G} if that player has 10 or less life. When you do, that player loses the game."));
+        //Ability ability = new DealsCombatDamageToAPlayerTriggeredAbility(new DoIfCostPaid(new EzioAuditoreDaFirenzeEffect(), new ManaCostsImpl<>("{W}{U}{B}{R}{G}")), true, true)
+        this.addAbility(
+                new ConditionalTriggeredAbility(
+                        new DealsCombatDamageToAPlayerTriggeredAbility(
+                                new DoIfCostPaid(
+                                        new LoseGameTargetPlayerEffect(),new ManaCostsImpl<>("{W}{U}{B}{R}{G}")
+                                ), false, true
+                        ),EzioAuditoreDaFirenzeWatcher::checkPermanent, "Whenever {this} deals combat damage to a player, you may pay {W}{U}{B}{R}{G} if that player has 10 or less life. When you do, that player loses the game."
+                )
+        );
+
+    }
+
+    private EzioAuditoreDaFirenze(final EzioAuditoreDaFirenze card) {
+        super(card);
+    }
+
+    @Override
+    public EzioAuditoreDaFirenze copy() {
+        return new EzioAuditoreDaFirenze(this);
+    }
+
+    public static Watcher makeWatcher(){
+        return new EzioAuditoreDaFirenzeWatcher();
+    }
```

Comment:
Other cards use this function so that when ~2-3 cards need the same watcher but it's not frequent enough to be a Common watcher, you can use `makeWatcher` to share it. Since nothing else uses this watcher, you can just add it to the ability directly and remove this `makeWatcher` until something else needs it.